src/test/java/uk/gov/hmcts/reform/migration/service/ApplicantDataMigrationServiceTest.java

```diff
@@ -0,0 +1,229 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.Applicant;
+import uk.gov.hmcts.reform.domain.OldApplicant;
+import uk.gov.hmcts.reform.domain.common.Address;
+import uk.gov.hmcts.reform.domain.common.EmailAddress;
+import uk.gov.hmcts.reform.domain.common.MobileNumber;
+import uk.gov.hmcts.reform.domain.common.ApplicantParty;
+import uk.gov.hmcts.reform.domain.common.TelephoneNumber;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ApplicantDataMigrationServiceTest {
+    private final String APPLICANT = "Applicant name";
+    private final String EMAIL = "email@email.com";
+    private final String MOBILE = "07778987656";
+    private final String COUNTY = "County";
+    private final String COUNTRY = "Country";
+    private final String POSTCODE = "BT17NT";
+    private final String POSTTOWN = "Ballymena";
+    private final String ADDRESSLINE1 ="ADDRESSLINE1";
+    private final String ADDRESSLINE2 = "ADDRESSLINE2";
+    private final String ADDRESSLINE3 = "ADDRESSLINE3";
+    private final String JOBTITLE = "JobTitle";
+    private final String TELEPHONE = "02825674837";
+    private final String PERSONTOCONTACT = "Person to contact";
+    private final String PARTYTYPE = "ApplicantParty type";
+    private final String LEADAPPLICANTINDICATOR = "Yes";
+    private final String PARTYID = UUID.randomUUID().toString();
+    private final String PBANUMBER= "PBA123456";
+
+    private final ApplicantDataMigrationService service = new ApplicantDataMigrationService();
+
+    @Test
+    void whenOldStructureDoesNotExistAcceptsShouldReturnFalse() {
+        CaseDetails caseDetails = CaseDetails.builder()
+            .id(1111L)
+            .data(ImmutableMap.of("data", "someData"))
+            .build();
+
+        assertThat(service.accepts().test(caseDetails)).isEqualTo(false);
+    }
+    @Test
+    void whenDataIsNullAcceptsShouldReturnFalse() {
+        CaseDetails caseDetails = CaseDetails.builder()
+            .id(1111L)
+            .data(null)
+            .build();
+
+        assertThat(service.accepts().test(caseDetails)).isEqualTo(false);
```

Comment:
Why you haven't added test case for accepted case? If logic was completely broken and always returned false your tests would not know about it. Please add it so there are no surprises.

Reply:
@swalker125 how would you test this?

Reply:
👍